packages/core/src/external/sftp/surescripts/command/connect.ts

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+import path from "path";
+import dotenv from "dotenv";
+dotenv.config({
+  path: path.resolve(__dirname, "../../../../../.env"),
+});
+
+import { Command } from "commander";
+import { SurescriptsSftpClient } from "../client";
+import { metriportBanner } from "./shared";
+const program = new Command();
+
+program
+  .description("Tests a connection to the Surescripts SFTP server")
+  .addHelpText("before", metriportBanner())
```

Comment:
while cool id avoid using the banner as it actually takes up many lines and I (and likely others) often use terminals that only show like 10 lines of context at a time lol 🙃

Reply:
Haha will do! I saw it on the other ones and couldn't resist 😄